AdminApi/src/AdminApi/Controllers/IdentitiesController.cs

```diff
@@ -112,6 +113,15 @@ public async Task<IActionResult> StartDeletionProcessAsSupport([FromRoute] strin
         var response = await _mediator.Send(new StartDeletionProcessAsSupportCommand(address), cancellationToken);
         return Created("", response);
     }
+
+    [HttpGet("{address}/DeletionProcess/{id}")]
+    [ProducesResponseType(typeof(GetDeletionProcessAsSupportQuery), StatusCodes.Status200OK)]
```

Comment:
I think that this annotation should be parametrized with the response type, not the query.